assets/src/block-editor/index.js

```diff
@@ -1,14 +1,14 @@
 // On running npm run dev, this will compile to assets/js/.
 
 /**
- * WordPress dependencies
+ * Internal dependencies
  */
-import { registerBlockType } from '@wordpress/blocks';
+import { registerBlocks } from './helpers';
 
-const blocks = require.context( './blocks', true, /(?<!test\/)index\.js$/ );
+const blocksToRegister = require.context(
+	'./blocks',
+	true,
+	/(?<!test\/)index\.js$/
+);
```

Comment:
No change here, I just moved each argument to a separate line to satisfy `npm run lint:js`